Mytinerary/Info.plist

```diff
@@ -20,6 +20,10 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSLocationAlwaysUsageDescription</key>
+	<string>This application requires location services to work</string>
```

Comment:
might be nice to still have the app work without location services enabled, it just won't show your current location